### ISSUE TITLE
View linked images in comments inline: oversized images not resized

### ISSUE THREAD

**Issue by fixer1234** [soscripted/sox#280]
### Current Behaviour
When oversized linked images are displayed inline in comments, they are not resized to fit in the space (column) reserved for the post, they are rendered at 100% of the image's native pixels.  This also causes the page to treat the comment area as if the image defines the column width for the comment text, as well.  

The result is that each comment becomes a single line of text disappearing off the right side of the screen, and the whole comment area becomes like a background behind other page content that is to the right of the post area.

Here is a smaple post: https://diy.stackexchange.com/questions/98751/wood-frame-not-on-foundation-and-in-contact-of-soil.  I had to zoom way out to capture the action of the image:

![image in comment](https://cloud.githubusercontent.com/assets/10065766/25409118/abaa743a-29de-11e7-93ce-3e4c5cc5b30c.png)

At normal magnification, here's what the bottom of the post and the top of the comments look like:

![image in comment1](https://cloud.githubusercontent.com/assets/10065766/25409174/eb99e54e-29de-11e7-9299-b6e1b05bfcf8.png)

This appears to simply be an oversight in the design, so I won't bother with console output.

---

### SOX Errors logged to the browser console -- F12 (if available)

---

### Steps to reproduce

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-fixedTopbar","Appearance-isQuestionHot","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-metaNewQuestionAlert","Appearance-scrollToTop","Appearance-standOutDupeCloseMigrated","Comments-autoShowCommentImages","Comments-commentReplies","Comments-confirmNavigateAway","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-enhancedEditor","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Sidebar-hideHireMe","Chat-renameChat","Voting-grayOutVotes","Voting-disableOwnPostVoteButtons","Extras-alwaysShowImageUploadLinkBox","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-warnNotLoggedIn","Extras-showMetaReviewCount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue"]

**Comment by shu8** [comment]
hmm doesn't happen in Chrome, must be a FF thing (again 😢 ) 

I'll try it out in FF ASAP :)

**Comment by fixer1234** [comment]
@shu8 I'm still on 2.0.26 (the version reported here).  Is there a newer version (and if so, a simple way to trigger an update)?

**Comment by shu8** [comment]
@fixer1234 try [dev 2.0.31](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)

**Comment by fixer1234** [comment]
@shu8 It's the same on 2.0.31.

**Comment by shu8** [comment]
Hmmm that's really weird :/ do you have any accessibility extensions
installed by any chance? :)

**Comment by fixer1234** [comment]
@shu8 none of those.

**Comment by fixer1234** [comment]
@shu8 I cranked up another system that still had 2.0.11 on it.  No issue on that one.  I updated to 2.0.31.  Still no issue.  That system has a pretty recent Firefox (53.0.3).  The one with the problem is an ESR version (I'll check the version and update this), but it's many generations of FF older (maybe 45).  So it's possible that it's related to the Firefox version.  I'll retest with another OS that has a recent version of FF and update this.
